### PR TITLE
bug:fix add assignees permission bug on New PR from forked repo

### PR DESCRIPTION
**Pull request by a1012112796**
In gitea now seting, only users have write permission can add Assignees, but if a user who don't have write permission also can add Assignees  when they creat a Pull Request from a forked repo , 
Because It haven't check the permission, so it's a bug . This PR should fix this bug by add a check for it. 

two other small changes:
* hide gear on new PR page when it's not necessary like #10750,
* remove some unusefull comments.

before change:

make PR frm a forked repo:
jta
------
after change:
fixed

### CODE REVIEW

File: templates/repo/issue/new_form.tmpl [hunk 1]
```diff
@@ -129,28 +135,6 @@
 						</a>
 					{{end}}
 				</div>
-
-			<!-- input id="assignee_ids" name="assignee_ids" type="hidden" value="{{.assignee_id}}">
```
Comment (a1012112796) [inline templates/repo/issue/new_form.tmpl]: Hello, I don't think these comments are usefull, so I suggest to remove them, If it have some special reason that should be kept. please tell me, I will recover them. thanks

File: templates/repo/issue/new_form.tmpl [hunk 2]
```diff
@@ -129,28 +135,6 @@
...
-				</div>
-			</div>
-			<div class="ui select-assignee list">
-				<span class="no-select item {{if .Assignee}}hide{{end}}">{{.i18n.Tr "repo.issues.new.no_assignees"}}</span>
-				<div class="selected">
-					{{if .Assignee}}
-						<a class="item" href="{{.RepoLink}}/issues?assignee={{.Assignee.ID}}"><img class="ui avatar image" src="{{.Assignee.RelAvatarLink}}"> {{.Assignee.Name}}</a>
-					{{end}}
-				</div>
-			</div>-->
```
Comment (a1012112796) [inline templates/repo/issue/new_form.tmpl:153]: Hello, I don't think these comments are usefull, so I suggest to remove them, If it have some special reason that should be kept. please tell me, I will recover them. thanks

**Comment by lafriks** [comment]
I don't think user with read permissions should be able to set assignee

**Comment by a1012112796** [comment]
> I don't think user with read permissions should be able to set assignee
Hello, this pr just make them as same, for permission question , it should have more discuss.

**Comment by a1012112796** [comment]
Hello, so, I add an extern check for it on new pull request page , Is something wrong about it?
@lafriks

**Comment by mrsdizzie** [comment]
I think having a gear you can't click is a bug but I think it is better to do what Github does and show some text like 'nothing to show' or 'no labels defined' etc... if there are no labels/milestones. Removing the gear creates the impression that you aren't allowed to add them and is the same interface for users who don't have permission to add them.

That could be for another PR if this one wants to just fix the write issue and remove the comment

**Comment by a1012112796** [comment]
> I think having a gear you can't click is a bug but I think it is better to do what Github does and show some text like 'nothing to show' or 'no labels defined' etc... if there are no labels/milestones. Removing the gear creates the impression that you aren't allowed to add them and is the same interface for users who don't have permission to add them.
> 
> That could be for another PR if this one wants to just fix the write issue and remove the comment

I'm sorry , but maybe you missunderstand me , the bug is reader of a repo should not have permission to add assignees,  but they can do it on New PR page now. the hide gear and delet 
are bounded small change. but one thing you are right , it's on New PR page we have make no 
distinction between having no permissions and having no options available for Labels, assignees, and Milestone, maybe It's necessary to make more check, so close this PR now. thanks for your suggestions.

**Pull request closed without merging**